packages/babel-helper-builder-react-jsx/README.md

```diff
@@ -18,11 +16,13 @@ require("@babel/helper-builder-react-jsx")({
   },
 
   pre: function (state: ElementState) {
-    // called before building the element
+    // function called with (state: ElementState) before building attribs
   },
 
   post: function (state: ElementState) {
-    // called after building the element
-  }
+    // function called with (state: ElementState) after building attribs
+  },
+
+  compat?: boolean // true if React is in compat mode
```

Comment:
Correct. IIRC, I just stuck this in there so it would throw an error if you're trying to use fragments with the `plugin-transform-react-jsx-compat` transform.